database/seedDefaultUser.js

```diff
@@ -0,0 +1,37 @@
+const crypto = require('crypto');
+const User = require('../models/User');
+const service = require('../services/user');
+const logger = require('../logger/logger');
+
+async function getKey(byteSize) {
+    let key = await crypto.randomBytes(byteSize);
+    return key;
+}
+
+const createDefaultUser = async (next) => {
+    try {
+        const userName = 'Admin';
+        const password = await getKey(12);
+        const data = { userName: userName, password: password.toString('hex') };
+        await service.create(data);
+        logger.info('Default User was created.');
+        return data;
+    } catch (error) {
+        next(error);
+    }
```

Comment:
no need to try catch everything, especially in that way.
Better use single middleware for errors 
https://dev.to/admirnisic/error-handling-in-node-js-with-express-part-2-30lb